_source/_posts/2022-04-28-gdpr-and-app-security.md

```diff
@@ -0,0 +1,148 @@
+---
+type: awareness
+github: https://github.com/oktadev/okta-dotnet6-webapi-example
+layout: blog_post
+title: "GDPR and application security"
```

Comment:
We should probably use a more descriptive title with keywords. Maybe use Keyword Planner and https://blog-title-analyzer.oktadev.dev to figure out a better title. Also, it should be title case. You can use this site for help. http://www.conissaunce.com/pages/infoq-title-case.html

Reply:
"The Benefits of GDPR for Application Security" scores very well!

Reply:
Nice. I'll update it.